nfc/include/nfc_t2t_lib.h

```diff
@@ -203,6 +209,7 @@ int nfc_t2t_payload_raw_set(const uint8_t *payload,
  *
  * @note When modifying the internal bytes, remember that they must be
  *	 consistent with the NFC hardware register settings
+ *	 (see @ref nfc_t2t_format_internal).
```

Comment:
Will this link work?

Reply:
Linking to functions does not currently work due to Breathe issue #438.